contracts/Context.json

```diff
@@ -478,8 +478,8 @@
     "version": "0.8.1+commit.df193b15.Emscripten.clang"
   },
   "networks": {},
-  "schemaVersion": "3.4.3",
-  "updatedAt": "2022-02-20T12:48:47.781Z",
+  "schemaVersion": "3.4.6",
+  "updatedAt": "2022-03-17T08:11:38.285Z",
```

Comment:
We should preserve latest updated at, but also highest version on these `/contracts` changes.